### PR TITLE
Approach for handling components: Passing conditions to the URL for m…

### PR DESCRIPTION
**Pull request by shabeeb-aot**
…anaging dissolution UI#11407 | changes for AssociationDetails component for SP

*Issue #:* /bcgov/entity#11407

*Description of changes:*
* Added Business Start Date in Dissolution component

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/Dissolution/AssociationDetails.vue [hunk 1]
```diff
@@ -134,6 +162,13 @@ export default class AssociationDetails extends Mixins(CommonMixin, EnumMixin) {
     return `${this.getCorpTypeDescription(this.getEntityType)}`
   }
 
+  /** The business start date. */
+  get businessStartDate (): string {
+    // SB TODO: get data from object and pass here
```
Comment (severinbeauvais) [inline src/components/Dissolution/AssociationDetails.vue:167]: When can you do this?

File: src/components/Dissolution/AssociationDetails.vue [hunk 2]
```diff
@@ -47,13 +46,28 @@
           <div v-else>Same as Mailing Address</div>
         </v-col>
       </v-row>
-    </div>
+    </article>
+
+    <template v-if="showBusinessDate">
+          <v-divider class="mx-6" />
```
Comment (severinbeauvais) [inline src/components/Dissolution/AssociationDetails.vue:52]: Fix indentation.

Also add a blank line after this (consistent with other HTML in here).

File: tests/unit/DocumentDelivery.spec.ts [hunk 3]
```diff
@@ -67,6 +67,23 @@ for (const mock of documentDeliveryCases) {
...
     })
 
+    it('displays additionalLabel labels', () => {
+      wrapper = shallowWrapperFactory(DocumentDelivery, {
+        additionalLabel: 'partners',
+        contactLabel: 'Registered Office'
+      })
+
+      expect(wrapper.findAll('label').at(2).text()).toBe('partners')
+    })
```
Comment (severinbeauvais) [inline tests/unit/DocumentDelivery.spec.ts:77]: This test should either be standalone (not in this "for" loop), or should use loop parameters to test different things for the items in `documentDeliveryCases[]`.

Also, please add a blank line between test methods.

File: tests/unit/DocumentDelivery.spec.ts [hunk 4]
```diff
@@ -67,6 +67,23 @@ for (const mock of documentDeliveryCases) {
...
+    it('displays additionalValue', () => {
+      wrapper = shallowWrapperFactory(DocumentDelivery, {
+        additionalValue: mock.tombstone.userEmail,
+        additionalLabel: 'partners',
+        contactLabel: 'Registered Office'
+      })
+      expect(wrapper.find('#additional-value').text()).toBe(mock.tombstone.userEmail)
+    })
+
```
Comment (severinbeauvais) [inline tests/unit/DocumentDelivery.spec.ts:86]: The easiest solution is to merge this and the previous unit test :)

**Review by severinbeauvais — COMMENTED** [review]
Remember to update the app version.

File: src/components/Dissolution/AssociationDetails.vue (hunk 1 above)
Comment (shabeeb-aot) [reply src/components/Dissolution/AssociationDetails.vue:167]: Once we have API available, will update the component, we have ticket on upcoming sprint

**Review by severinbeauvais — APPROVED** [review]
LGTM

File: package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "3.5.0",
+  "version": "3.5.1",
```
Comment (severinbeauvais) [inline package.json:3]: Main branch is already at 3.5.1. Please change this to 3.5.2 (or ignore if it doesn't matter to you, this time, to track the version in the deployed app).

**Comment by severinbeauvais** [comment]
@cameron-freshworks or @saravanpa-aot , do you have time to review this PR? If not, lmk and we'll proceed accordingly.

**Pull request merged**